frontend/packages/topology/integration-tests/features/topology/quick-search-topology.feature

```diff
@@ -82,5 +82,36 @@ Feature: Provide quick search from topology/list views to add to project
              When user clicks Add to project button
               And user enters "monitoring" in Add to project search bar
               And user clicks on View all Quick Starts option
-             Then user is taken to the search results in context of the Quick Starts Catalog.
+             Then user is taken to the search results in context of the Quick Starts Catalog
 
+
+        @odc-22 @to-do
+        Scenario: Quick Add of Devfile in topology graph view: T-02-TC011
+            Given user is at topology graph view
+             When user clicks Add to project button
+              And user enters "node" in Add to project search bar
+              And user selects Basic NodeJS Devfiles option
+              And user clicks on Create Application
+              And user clicks on Create button in the Import from Devfile page
+             Then user is taken to the Topology page with "devfile-sample-git" workload created
+
+
+        @odc-22 @to-do
+        Scenario: Quick Add of Devfile Sample in topology graph view: T-02-TC012
+            Given user is at topology graph view
+             When user clicks Add to project button
+              And user enters "node" in Add to project search bar
+              And user selects Basic NodeJS Samples option
+              And user clicks on Create Devfile Samples
+              And user assigns Name as "basic-nodejs-sample-ex1"
+              And user clicks on Create button in Import from Devfile page
+             Then user is taken to the Topology page with workload "basic-nodejs-sample-ex1" created
+
+
+        @to-do
```

Comment:
Please add @odc-22, so we get to know. this is related to that particular epic

Reply:
Added